test/parallel/test-webcrypto-export-import-ec.js

```diff
@@ -262,6 +262,35 @@ async function testImportJwk(
   }
 }
 
+async function testImportRaw({ name, publicUsages }, namedCurve) {
+  const jwk = keyData[namedCurve].jwk;
+
+  const [publicKey] = await Promise.all([
+    subtle.importKey(
+      'raw',
+      Buffer.concat([
+        Buffer.alloc(1, 0x04),
+        Buffer.from(jwk.x, 'base64url'),
+        Buffer.from(jwk.y, 'base64url'),
+      ]),
```

Comment:
Side note... It would be nice if Web Crypto supported using `Blob` for this kind of stuff.